pep-0261.txt

```diff
@@ -5,280 +5,287 @@ Last-Modified: $Date$
 Author: Paul Prescod <paul@prescod.net>
 Status: Final
 Type: Standards Track
+Content-Type: text/x-rst
 Created: 27-Jun-2001
 Python-Version: 2.2
 Post-History: 27-Jun-2001
 
 
 Abstract
+========
 
-    Python 2.1 unicode characters can have ordinals only up to 2**16 -1.  
-    This range corresponds to a range in Unicode known as the Basic
-    Multilingual Plane. There are now characters in Unicode that live
-    on other "planes". The largest addressable character in Unicode
-    has the ordinal 17 * 2**16 - 1 (0x10ffff). For readability, we
-    will call this TOPCHAR and call characters in this range "wide 
-    characters".
+Python 2.1 unicode characters can have ordinals only up to 2**16 -1.
+This range corresponds to a range in Unicode known as the Basic
+Multilingual Plane. There are now characters in Unicode that live
+on other "planes". The largest addressable character in Unicode
+has the ordinal 17 * 2**16 - 1 (0x10ffff). For readability, we
+will call this TOPCHAR and call characters in this range "wide
+characters".
 
 
 Glossary
-
-    Character 
-        
-        Used by itself, means the addressable units of a Python 
-        Unicode string.
-
-    Code point
-
-        A code point is an integer between 0 and TOPCHAR.
-        If you imagine Unicode as a mapping from integers to
-        characters, each integer is a code point. But the 
-        integers between 0 and TOPCHAR that do not map to
-        characters are also code points. Some will someday 
-        be used for characters. Some are guaranteed never 
-        to be used for characters.
-
-    Codec
-
-        A set of functions for translating between physical
-        encodings (e.g. on disk or coming in from a network)
-        into logical Python objects.
-
-    Encoding
-
-        Mechanism for representing abstract characters in terms of
-        physical bits and bytes. Encodings allow us to store
-        Unicode characters on disk and transmit them over networks
-        in a manner that is compatible with other Unicode software.
-
-    Surrogate pair
-
-        Two physical characters that represent a single logical
-        character. Part of a convention for representing 32-bit
-        code points in terms of two 16-bit code points.
-
-    Unicode string
-
-          A Python type representing a sequence of code points with
-          "string semantics" (e.g. case conversions, regular
-          expression compatibility, etc.) Constructed with the 
-          unicode() function.
+========
+
+Character
+   Used by itself, means the addressable units of a Python
```

Comment:
This becomes a `definition list` item: http://cpython-devguide.readthedocs.io/documenting.html?highlight=definition%20list#lists-and-quotes

But I can convert these into sub-headings too.

<img width="831" alt="screen shot 2017-02-10 at 11 07 07 am" src="https://cloud.githubusercontent.com/assets/5844587/22840143/3d565432-ef81-11e6-82e6-81ad6703b21a.png">